languages/python/exercises/concept/none/.docs/instructions.md

```diff
@@ -0,0 +1,89 @@
+In this exercise, you will get the basic idea of how to use 'None'.
+
+The exercise has 6 challenges to complete.
+
+You are the Maître D' of a hotel, Your tasks are given based on the number of people who are waiting to eat at your hotel. Complete the tasks accordingly and proceed.
+
+## 1. Arrange the Seats
+
+Define the `arrange_the_seats()` function with 1 parameter. You know the number of guests who have reserved a table today.
+
+The task of this function is to return a `dict` object containing the seat number (in `int`) as the key and `None` as the value for the number of guests who are going to attend.
+
+```python
+arrange_the_seats(guests=4)
+#=> {1: None, 2: None, 3: None, 4: None}
+```
+
+## 2. Is the Seat Empty?
+
+Define the `is_seat_empty()` function that takes 1 parameter. First parameter will contain the value of the seat which is checked before assigning to a guest.
+
+If the seat is empty, It will be of `None` value, else it will be mentioned with string `Occupied`.
+
+If the seat is empty, return `True` else return `False`.
+
+```python
+is_seat_empty(seat_value="Occupied")
+# => False
+is_seat_empty(seat_value=None)
+# => True
+```
+
```

Comment:
```suggestion
## 3.  Find All the "Empty" Seats

Define the `find_all_available_seats(seats)` function that takes 1 parameter (_a seating chart dictionary_) and returns a `list` of seat numbers that are available for guests that are currently waiting.

If a seat is empty, It will be of `None` value in the dictionary.   Occupied seats will have the name of the guest.

```python
find_all_available_seats(seats)
>>>[1,16,5,7,3]
```